compiler-rt/lib/fuzzer/FuzzerUtilFuchsia.cpp

```diff
@@ -607,7 +607,11 @@ size_t PageSize() {
 }
 
 void SetThreadName(std::thread &thread, const std::string &name) {
-  // TODO ?
+  if (name.size() > 31) {
+    name.resize(31);
+  }
```

Comment:
This is just a nit, but we don't use curly braces for single statements.
```suggestion
  if (name.size() > 31)
    name.resize(31);
```